doc/source/analytics/analytics.md

```diff
@@ -83,6 +83,12 @@ For this purpose you can supply extra fields in the returned meta data of the re
 				"key": "mygauge",
 				"value": 22.0
 			},
+			{
+				"type": "HISTOGRAM",
```

Comment:
Would be nice to add one item to list below too.

Reply:
Could you clarify a bit more what you mean?

Reply:
A bullet point entry for HISTOGRAM here
```
We provide three types of metric that can be returned in the meta.metrics list:

COUNTER : a monotonically increasing value. It will be added to any existing value from the metric key.
GAUGE : an absolute value showing a level, it will overwrite any existing value.
TIMER : a time value (in msecs), it will be aggregated into Prometheus' HISTOGRAM.
```

Reply:
Oh, and it seems there is also a proto snippet just next paragraph.

Reply:
I updated the doc in it